src/app/loadout-builder/process.ts

```diff
@@ -331,6 +334,27 @@ export function generateMixesFromPerks(
   return mixes;
 }
 
+function getBaseStatValue(stat: DimStat, item: DimItem) {
+  let baseStatValue = stat.value;
+
+  // Checking energy tells us if it is Armour 2.0
+  if (item.isDestiny2() && item.sockets && item.energy) {
+    for (const socket of item.sockets.sockets) {
+      if (socket.plug && socket.plug.plugItem.investmentStats) {
+        for (const plugStat of socket.plug.plugItem.investmentStats) {
```

Comment:
This is technically incorrect, but works because armor stats are not currently scaled (see https://github.com/DestinyItemManager/DIM/wiki/DIM's-New-Stat-Calculations).

Reply:
That being the case, would it be better to calculate this here or when the stats are initially generated? 

I initially did it on generation but it meant I had to add some values that seemed a little out of place on the DimStat model. The positive would mean that I can simplify the logic here by just adding or subtracting a mod value.

Reply:
As another solution I could do something similar to the masterwork stat handing and a item.modInfo section to DimItem

Reply:
Hang on I just figured out that the calculated plug stats are already there, so I will use those